uwblueprint/website-bp
# Implement Export CSV Button

This PR seeks to implement the export CSV button feature. When the button is clicked, a CSV file of the information displayed on the page is automatically downloaded. This feature is achieved using the react-csv library. More information about the library can be found [here](https://www.npmjs.com/package/react-csv). 

![image](https://user-images.githubusercontent.com/63923219/204160161-82c0392e-8102-4545-9205-a5e2abaefd61.png)
![image](https://user-images.githubusercontent.com/63923219/204160166-62de111d-9f81-4b1a-bb1b-228960acc4c5.png)
![image](https://user-images.githubusercontent.com/63923219/204160185-ba5f2e8d-0e36-4ef7-86a7-07df89b675f6.png)




### Steps to test

1.Pull in this branch to your local.
2. Run yarn to install and update the new libraries used in implementing this feature.
3. Run on your localhost, and sign into the admin portal.
4. Click on the Export CSV button. The CSV file of the student's application should automatically download with a default file name of 'export.csv'.
5. Open the file, and confirm the information matches the one displayed on the webpage. 



### What should reviewers focus on?

- The Export CSV button should work as expected.

### Checklist
